index.html

```diff
@@ -329,6 +338,12 @@
         that is the <a>context object</a>, replace the <a>context object</a>'s
         `options` with <var>options</var>.
         </li>
+        <li>If _options'_ <a for="PerformanceObserverInit">buffered</a>
+          attribute is present with a value of `true`, for each entryType of
+          the `entryTypes` sequence, copy the corresponding entries from the
+          global performance timeline to the newly <a>registered performance
```

Comment:
> copy the corresponding entries from the global performance timeline

We need to spell out the steps here. We're running the same operation as `getEntriesByType` [1], so we can reuse the language from that. Something like:

- For each type X in ...
  - Let _entries_ be PerformanceEntryList object returned by 4.2. Filter buffer by name and type algorithm with buffer set to performance entry buffer, name set to null, and type set to X.
  - Append entries to `context object`'s `observe buffer`.

[1] https://w3c.github.io/performance-timeline/#getentries-method